daemon/prune.go

```diff
@@ -428,7 +432,8 @@ func (daemon *Daemon) NetworksPrune(ctx context.Context, pruneFilters filters.Ar
 	select {
 	case <-ctx.Done():
 		logrus.Warnf("NetworksPrune operation cancelled: %#v", *rep)
-		return nil, ctx.Err()
+		// when the context is cancelled, return current report struct and nil
```

Comment:
This is obvious from the return line, I don't think we need the comment explaining